conda/resolve.py

```diff
@@ -816,7 +816,11 @@ def bad_installed(self, installed, new_specs):
         groups, trackers = build_groups(dists)
         C = self.gen_clauses(groups, trackers, specs)
         constraints = self.generate_spec_constraints(C, specs)
-        solution = C.sat(constraints)
+        try:
+            solution = C.sat(constraints)
+        except:
```

Comment:
I understand that's what we should do, yes. But this one is proving very hard to reproduce, and it is preventing people from updating their environments.